actionpack/lib/action_controller/railtie.rb

```diff
@@ -69,5 +69,13 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "action_controller.request_forgery_protection" do |app|
+      ActiveSupport.on_load(:action_controller) do
```

Comment:
This is also going to run for `ActionController::Api`, so it is better to use `on_load(:action_controller_base` here. That way we can do:

```ruby
ActiveSupport.on_load(:action_controller_base) do
  if app.config.action_controller.default_protect_from_forgery
    protect_from_forgery with: :exception
  end
end
```